firebase-database/src/androidTest/java/com/google/firebase/database/FirebaseDatabaseTest.java

```diff
@@ -20,8 +20,8 @@
 import static org.junit.Assert.assertSame;
 import static org.junit.Assert.fail;
 
-import androidx.test.InstrumentationRegistry;
-import androidx.test.runner.AndroidJUnit4;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.platform.app.InstrumentationRegistry;
```

Comment:
This library has a breaking change; you now need to access a singleton with `getInstrumentation()` before getting contexts.